src/transformer_util.ts

```diff
@@ -124,28 +124,19 @@ function prepareNodesBeforeTypeScriptTransform(context: ts.TransformationContext
         }
       }
 
-      let importOrReexportNode: (ts.ExportDeclaration|ts.ImportDeclaration)&
-          {moduleSpecifier: ts.StringLiteral}|undefined;
-      if (node.kind === ts.SyntaxKind.ImportDeclaration) {
-        const id = node as ts.ImportDeclaration;
+      if (node.kind === ts.SyntaxKind.ImportDeclaration ||
+          node.kind === ts.SyntaxKind.ExportDeclaration) {
+        const ied = node as ts.ImportDeclaration | ts.ExportDeclaration;
         // this should always be a StringLiteral, see the typescript docs.
-        if (id.moduleSpecifier.kind !== ts.SyntaxKind.StringLiteral) {
-          throw new Error(`Unexpected moduleSpecifier kind: ${id.moduleSpecifier.kind}`);
-        }
-        importOrReexportNode = node as ts.ImportDeclaration & {moduleSpecifier: ts.StringLiteral};
-      } else if (node.kind === ts.SyntaxKind.ExportDeclaration) {
-        const ed = node as ts.ExportDeclaration;
-        if (ed.moduleSpecifier) {
+        if (ied.moduleSpecifier) {
           // this should always be a StringLiteral, see the typescript docs.
```

Comment:
This comment is duplicated from two lines up, I think maybe delete the upper one?